spec/double_entry/reporting/line_aggregate_filter_spec.rb

```diff
@@ -49,7 +49,7 @@
           # an example of providing a single metadatum criteria to filter on
           {
             :metadata => {
-              :meme => :business_cat,
+              :meme => 'business_cat',
```

Comment:
No, but the tests should illustrate typical usage. I believe we had decided that generally the keys would be symbols and the values would be strings.